internal/operands/template-validator/resources.go

```diff
@@ -217,11 +213,9 @@ func newValidatingWebhook(namespace string) *admission.ValidatingWebhookConfigur
 					Resources:   []string{"virtualmachines"},
 				},
 			}},
-			FailurePolicy: &fail,
-			SideEffects:   &sideEffectsNone,
-			// TODO - add "v1" to the list once the template-validator
-			//        is updated to new API
-			AdmissionReviewVersions: []string{"v1beta1"},
+			FailurePolicy:           &fail,
+			SideEffects:             &sideEffectsNone,
+			AdmissionReviewVersions: []string{"v1"},
```

Comment:
Is it safe to drop the `v1beta1` or should we keep it for this release (any potential upgrade problems)?

Reply:
Ah, nvm. iiuc it's just the version we process/send, right?

Reply:
Yes, it is the version sent by k8s api server to our webhook. I'm not sure which k8s version introduced `v1` admissionregistration version, but it is present in `v1.17` and later.
https://v1-17.docs.kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/